packages/models-library/src/models_library/clusters.py

```diff
@@ -161,16 +173,29 @@ class Config(BaseCluster.Config):
             ]
         }
 
-    @validator("access_rights", always=True, pre=True)
+    @root_validator(pre=True)
     @classmethod
-    def check_owner_has_access_rights(cls, v, values):
+    def check_owner_has_access_rights(cls, values):
+        if values["id"] == "default":
+            return values
         owner_gid = values["owner"]
         # check owner is in the access rights, if not add it
-        if owner_gid not in v:
-            v[owner_gid] = CLUSTER_ADMIN_RIGHTS
+        access_rights = values.get("access_rights", values.get("accessRights"))
+        if owner_gid not in access_rights:
+            access_rights[owner_gid] = CLUSTER_ADMIN_RIGHTS
         # check owner has full access
-        if v[owner_gid] != CLUSTER_ADMIN_RIGHTS:
+        if access_rights[owner_gid] != CLUSTER_ADMIN_RIGHTS:
             raise ValueError(
-                f"the cluster owner access rights are incorrectly set: {v[owner_gid]}"
+                f"the cluster owner access rights are incorrectly set: {access_rights[owner_gid]}"
             )
-        return v
+        return values
+
+
+class DefaultCluster(Cluster):
+    """the default cluster is usable by everyone as USER"""
+
+    id: Literal["default"] = "default"
+    name: Literal["Default cluster"] = "Default cluster"
+    type: ClusterType = ClusterType.ON_PREMISE
+    owner: GroupID = 1
```

Comment:
Maybe add a comment here explaining that this is the linux user group (correct?)